google/cloud/spanner_dbapi/cursor.py

```diff
@@ -305,14 +305,19 @@ def fetchone(self):
         self._raise_if_closed()
 
         try:
-            return next(self)
+            res = next(self)
```

Comment:
I assume that the current implementation is mainly for adding the checksum feature, and not any retry logic.

But once we start adding retry logic, it is important to realize that this call to `next(self)` can cause an `Aborted` error which then again should initiate a transaction retry.

Reply:
Yes, I'm pusing retry logic only today.